lib/protractor.js

```diff
@@ -440,12 +440,21 @@ Protractor.prototype.get = function(destination, opt_timeout) {
 
   destination = this.baseUrl.indexOf('file://') === 0 ?
     this.baseUrl + destination : url.resolve(this.baseUrl, destination);
+  this.driver.wait(function() {
```

Comment:
Why does this need to be a wait? We could just do:
`destinationPromise = self.plugins_.mutatUrl(destination);`

and then below

`return destinationPromise.then(function(destination) {
  return this.driver.get(destination)....
`